consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeMiningCoordinator.java

```diff
@@ -71,6 +71,14 @@ default boolean isCompatibleWithEngineApi() {
    */
   BlockProcessingResult validateBlock(final Block block);
 
+  /**
+   * Validate a proposed block.
+   *
+   * @param block the block
+   * @return the block processing result
+   */
+  BlockProcessingResult validateProposedBlock(final Block block);
```

Comment:
Does this method need to be defined on the interface? Looks like would only ever be needed internally as part of preparePayload?

Reply:
Good catch. I think the same might be true for the existing validateBlock method actually, looks like it could be private. 
Maybe there's something weird about the TransitionCoordinator needing it 🤷

Reply:
The TransitionCoordinator doesn't "need" this method, I added the method to the interface mainly because I understood we're not dropping the Transition related code so it probably make sense to keep it updated/compatible with latest changes, specially if the cost is as low as a simple delegation.

Reply:
I couldn't see any code actually calling the TransitionCoordinator version of the method...though maybe it would become clear if you tried to remove it.

Reply:
All good, done

Reply:
Done